cmd/controller/run.go

```diff
@@ -163,11 +163,7 @@ func Run(opts Options, runLog logr.Logger) {
 
 	{ // add controller for PackageInstall
 		pkgInstallCtrlOpts := controller.Options{
-			Reconciler: &PackageInstallReconciler{
-				kcClient:  kcClient,
-				pkgClient: pkgClient,
-				log:       runLog.WithName("ipr"),
-			},
+			Reconciler:              NewPackageInstallReconciler(kcClient, pkgClient, coreClient, runLog.WithName("ipr")),
```

Comment:
do we want to update the arg here to `WithName("PackageInstallReconciler")` ?   or `pir` ?

Reply:
Yes, but maybe outside this pr preferably.

Reply:
ok

Reply:
dont want to have noisy logs with PackageInstallReconciler on every single line.